modules/serverless-installing-cli-linux-ibm-z_tarball.adoc

```diff
@@ -0,0 +1,39 @@
+[id="installing-cli-linux_ibm_z_tarball_{context}"]
+
+== Tarball Installation
```

Comment:
These should all be H1 headings in the modules, i.e. `=` instead of `==`

Reply:
The main headings are `Installing the `kn` CLI for Linux on IBM Power Systems` and `Installing the `kn` CLI for Linux on IBM Z` , Tarball Installation and RPM Installation are the sub headings, correct me if I am wrong, so these should be H2

Reply:
The title for a module always gets a H1, the [leveloffset=+1] used when including it in the assembly makes it appear as a H2 _in the assembly._
Please follow the modular doc guidelines https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

For additional guidance on your structuring please reach out to Moushumi Kalita or @vikram-redhat for OpenShift specific issues.